README.rst

```diff
@@ -199,7 +199,7 @@ Everything is working normally.
 ~~~~~~~~~~~
 
 One of the metrics has a value which is not in the expected range
-(threshold value crossed).
+(AlertSettings value crossed).
```

Comment:
applies to here as well, see how I have edited it in the branch where this will be merged